organization/settings.py

```diff
@@ -729,3 +734,18 @@
 ARTICLE_KEYWORDS = ['', ]
 
 DATA_UPLOAD_MAX_NUMBER_FIELDS = 10000
+
+# OAUTH2
+OAUTH2_IRCAM = True
+
+OAUTH_SERVER_BASEURL = 'https://auth.ircam.fr/'
+USER_SERVER_BASEURL = 'https://auth.ircam.fr/'
+#OAUTH_SERVER_BASEURL = 'http://my-oauth2-server.dev:8000'
+#USER_SERVER_BASEURL = 'http://localhost:8130'
+
+LOGOUT_URL = USER_SERVER_BASEURL + '/accounts/logout/'
```

Comment:
What if OAuth2 login is disabled ?

Reply:
The logout is identical in the both cases. 
But in the case of Oauth2, the user stays connected to the auth server, allowing a "one click" reconnection in Mezzo.